src/tiledimage.js

```diff
@@ -707,7 +705,7 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
         };
 
         try {
-            if (!isArray(polygons)) {
+            if ($.isArray(polygons)) {
```

Comment:
You've lost the `!` here... otherwise it all looks good!

Reply:
Oops